### PR TITLE
filter errata by type, severity etc.

### PR DESCRIPTION
**Pull request by blayson**

### CODE REVIEW

File: webapp/webapp.spec.yaml [hunk 1]
```diff
@@ -741,6 +741,23 @@ components:
...
+            - type: string
+              example: 'security'
+            - type: array
+              items:
+                type: string
+                example: security
+        severity:
+          oneOf:
+            - type: string
+              example: 'Critical'
```
Comment (jdobes) [inline webapp/webapp.spec.yaml]: shouldn't this be nullable as well?

Comment (blayson) [reply webapp/webapp.spec.yaml]: hmm, I fixed it
but I have tested it before and it was working fine 🤷‍♂

Comment (jdobes) [reply webapp/webapp.spec.yaml]: are you testing it with upgraded schema (null severity_id columns in errata table)? and current webapp dump (generated by reposcan)?

**Comment by jdobes** [comment]
any update?

**Pull request merged**